docs/DeveloperGuide.md

```diff
@@ -1423,6 +1459,53 @@ testers are expected to do more *exploratory* testing.
 
 1. _{ more test cases …​ }_
 
+### Creating a consultation
+
+1. Creating a consultation with specified date, time and student(s).
+
+    1. Prerequisites: Date and time inputs must be valid and student names must exist in address book. There must not be a duplicate consultations (same date, time and students).
```

Comment:
It should be "... a duplicate consultation"

Reply:
I think duplicate consultations.

Reply:
No, you're right!